package.json

```diff
@@ -24,15 +24,14 @@
   ],
   "dependencies": {
     "incremental-dom": "0.4.1",
-    "window-or-global": "^1.0.1"
+    "window-or-global": "1.0.1"
   },
   "devDependencies": {
-    "birdpoo": "0.x",
-    "document-register-element": "1.x",
-    "react": "15.x",
-    "react-dom": "15.x",
-    "skatejs-build": "9.x",
-    "skatejs-web-components": "2.0.2"
+    "birdpoo": "0",
```

Comment:
I remember this discussion, that `1.x` is confusing for some people, everyone is used to `^1.0.0`. Don't you think this would be even more confusing?